libyul/backends/wasm/WordSizeTransform.cpp

```diff
@@ -81,8 +81,8 @@ void WordSizeTransform::operator()(ForLoop& _for)
 {
 	(*this)(_for.pre);
 	_for.condition = make_unique<Expression>(FunctionCall{
-		locationOf(*_for.condition),
-		Identifier{locationOf(*_for.condition), "or_bool"_yulstring},
+		debugDataOf(*_for.condition),
```

Comment:
The rest of the file just uses `.debugData`, perhaps do it in these two lines too?